examples/vumps/vumps_ising.jl

```diff
@@ -7,6 +7,33 @@ include(
   ),
 )
 
+function calc_vN_entropy(S)
+  SvN, norm = 0.0, 0.0
+  for n in 1:dim(S, 1)
+    p = S[n, n]^2
+    SvN -= p * log(p)
+    norm += p
+  end
+  return SvN, norm
+end
+
+function entropy_finite(ψ, b)
+  ψ = orthogonalize(ψ, b)
+  U, S, V = svd(ψ[b], (linkind(ψ, b - 1), siteind(ψ, b)))
+  SvN, norm = calc_vN_entropy(S)
+  @assert norm ≈ 1.0
+  return SvN
+end
+
+function entropy_infinite(ψ, b)
+  #calculate entropy
+  C = ψ.C[b]
+  Ũ, S, Ṽ = svd(C, inds(C)[1])
+  SvN, norm = calc_vN_entropy(S)
+  @assert norm ≈ 1.0
+  return SvN
+end
```

Comment:
We shouldn't define these in multiple files. Better to make a common file with one set of definitions, maybe in `ITensorInfiniteMPS.jl/examples/vumps/src/entropy.jl`. Repeating the same code multiple times adds maintenance burden.

Reply:
Of course it would be better to have common sets of entropy definitions somewhere in the ITensor library which could be used here but that is a larger issue, not ready to commit to that yet before thinking it through.